Dinosaur.R

```diff
@@ -0,0 +1,2 @@
+x <- c("Brontosaurus", "Trex", "Triceratops")
+sample(x, 1)
```

Comment:
Looks good to me, but I am no expert with functions, so @nickhaf, please double ckeck!